src/status_im/contexts/wallet/swap/select_asset_to_pay/view.cljs

```diff
@@ -26,7 +26,22 @@
                                          {:token    token
                                           :network  (when (= (count token-networks) 1)
                                                       (first token-networks))
-                                          :stack-id :screen/wallet.swap-select-asset-to-pay}])))]
+                                          :stack-id :screen/wallet.swap-select-asset-to-pay}])
+                           (rf/dispatch [:wallet.swap/select-asset-to-receive {:token token}])
+                           (rf/dispatch [:wallet.swap/set-pay-amount 100])
+                           (rf/dispatch [:wallet.swap/set-swap-proposal
+                                         {:from           {:chain-id               1
+                                                           :native-currency-symbol "ETH"}
+                                          :to             {:chain-id               1
+                                                           :native-currency-symbol "ETH"}
+                                          :gas-amount     "23487"
+                                          :gas-fees       {:base-fee                 "32.325296406"
+                                                           :max-priority-fee-per-gas "0.011000001"
+                                                           :eip1559-enabled          true}
+                                          :receive-amount 99.98
+                                          :receive-token  {:symbol  "SNT"
+                                                           :address "0x432492384728934239789"}}])
```

Comment:
Dummy data, will be removed later

Reply:
can we at least move all dummy data into a def?

Reply:
Sure, will do!